web/docs/understand/operators/transformations/enumerate.md

```diff
@@ -0,0 +1,49 @@
+# enumerate
+
+Prepend a column with row numbers.
+
+## Synopsis
+
+```
+enumerate [-f|--field=<string>]
+```
+
+## Description
+
+The `enumerate` operator prepends a new column with row numbers to the beginning
+of the input record.
+
+The operator counts row numbers per schema separately.
+
+### `-f|--field=<string>`
+
+Sets the name of the output field.
+
+Defaults to `#` in order to avoid conflicts with existing field names.
```

Comment:
I would argue that the default should be `"index"` to align with most programming languages.

Reply:
`enumerate index` does this now for you. I'd rather be very conservative with the field names initially, `#` is almost _never_ a clash.